README.md

```diff
@@ -8,6 +8,16 @@ The charter for OSS Review Toolkit (ORT) can be found at:
 
 https://github.com/oss-review-toolkit/ort-governance/blob/master/CHARTER.md
 
+### Technical Steering Committee
+
+As per ORT's charter, the project has a Technical Steering Committee (TSC) made up of the following members:
```

Comment:
Should we add "(in alphabetical order)" like [here](https://github.com/oss-review-toolkit/ort-governance/blob/43a03a87d6946e8469d9d6c67468869cd2246288/CONTRIBUTING.md?plain=1#L5C13-L5C36)?